packages/npm-dependencies-validation/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "@sap-devx/npm-dependencies-validation",
+  "version": "1.3.0",
+  "private": true,
+  "description": "",
+  "author": "SAP SE",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/SAP/app-studio-toolkit"
+  },
+  "main": "./dist/src/index",
+  "types": "./dist/src/index.d.ts",
```

Comment:
we should consider if we want separate types declarations or re-use the generated types signatures (I am not sure...)

Reply:
This comment is about the issue of api.ts (`export {x,y,z} from "..."`) we discussed in voice.